wscript

```diff
@@ -482,6 +484,12 @@ def build(ctx):
 			use_extra.append('HDF5')
 			use_extra.append('lib/julea-kv')
 			use_extra.append('lib/julea-object')
+		elif client == 'hdf5-db':
+			use_extra.append('HDF5DB')
```

Comment:
What is this?

Reply:
the db and kv Julia vol plugins are compiled into two independent libraries next to each other. the old 'hdf' library could be renamed to 'hdf-kv'. I did not rename the old library, to reduce merge conflicts.